content_script.js

```diff
@@ -0,0 +1,82 @@
+var filteredTags = ["script", "noscript", "style"];//, "link"];
```

Comment:
Global constants should be written like FILTERED_TAGS. I forgot to mention it before, but please adhere to the
Google JavaScript style guide:

https://google.github.io/styleguide/javascriptguide.xml